monkey/monkey_island/cc/services/attack/technique_reports/T1154.py

```diff
@@ -0,0 +1,37 @@
+from monkey_island.cc.services.attack.technique_reports import AttackTechnique
+from monkey_island.cc.database import mongo
+from common.utils.attack_utils import ScanStatus
+from common.data.post_breach_consts import POST_BREACH_TRAP_COMMAND
+
+
+__author__ = "shreyamalviya"
+
+
+class T1154(AttackTechnique):
+    tech_id = "T1154"
+    unscanned_msg = "Monkey did not use the trap command on the system."
```

Comment:
Two seperate issues:

One, "on the system" is incorrect, the Monkey did not use the command at all (on `n` systems - might have been 100 computers). So the phrasing here is wrong.

Two, and this is about this and all other techniques we implement using PBAs. We should add a way (link to documentation/hardcoded message) for the users to know that the techniques might have not been executed because it's not configured as a PBA, and how to activate it if they DO want it. This message doesn't really explain WHY the technique wasn't used... For example, two better possible messages:

- Monkey did not use the `trap` command because there were no Linux machines in this attack.
- Monkey did not use the `trap` command because the relevant Post Breach Action is disabled. see this documentation page to learn how to activate it. 

What do you think?

Reply:
(1) Okay, will change!
(2) Oh yeah, I think that'd be much better! So basically, for every technique's unscanned message, we check in the configuration if it's disabled or not and then show the message accordingly?

Reply:
Does it even show reports for deselected techniques? I think a better solution would be not to even show the report if technique wasn't selected.

Reply:
@VakarisZ I disagree, I'd really prefer for users to KNOW that we are missing out on stuff
@shreyamalviya start a discussion over on #mitre slack, present 2-3 options (including cost as well), and let's decide between the options you'll present.